app/src/main/java/com/codepath/apps/restclienttemplate/TimelineActivity.java

```diff
@@ -41,16 +45,33 @@
     SwipeRefreshLayout swipeContainer;
     EndlessRecyclerViewScrollListener scrollListener;
     MenuItem miActionProgressItem;
+    FloatingActionButton fab;
```

Comment:
This is not a good name, you have the prefix but not the actual name (to show function) of the button